prelude/__internal__/String.mad

```diff
@@ -7,3 +7,9 @@ replace :: String -> String -> String -> String
 export replace = (regex, replacing, input) => (#-
   input.replace(new RegExp(regex), replacing)
 -#)
+
+split :: String -> String -> List String
+export split = (separator, str) => (#- str.split(separator) -#)
+
+lines :: String -> List String
+export lines = split("\n")
```

Comment:
Potentially also `unlines` ? `words` / `unwords`?

Reply:
Potentially. Well the Haskell name for it is lines though, but happy to go with anything else. That was mostly used to work on the lcov parser.